source/core/src/main/com/csse3200/game/components/inventory/InventoryDisplayManager.java

```diff
@@ -56,6 +59,13 @@ public void updateDisplays() {
 
         int displayCount = openInventoryDisplays.size();
 
+        //Play the inventory sound effect
+        try {
+            ServiceLocator.getSoundService().getEffectsMusicService().play(EffectSoundFile.INVENTORY_OPEN);
```

Comment:
This should be in the Display rather than the DisplayManager. This means that all inventoryDisplays will play the same sound, rather we want it so that each display can play a specified sound

Reply:
Ahh okay that makes sense, I thought it would be for all chest/inventory I'll edit that now

Reply:
In that case should I move it to the toggleOpen function of the inventory display?